plugins/modules/inventory_intent.py

```diff
@@ -28,6 +28,10 @@
     description: Set to True to verify the Cisco DNA Center config after applying the playbook config.
     type: bool
     default: False
+  dnac_log_level:
+    description: Log levels are used to categorize the logs based on their severity.
+    type: str
+    default: INFO
```

Comment:
How the customers know other options?
Can we specify the choices?
    choices: [ CRITICAL, ... INFO,... ]

Reply:
added.